Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -334,6 +335,8 @@ export interface QTestArguments extends Transformer.RunnerArguments {
     qTestBuildType?: string;
     /** Specifies the environment variables to forward to qtest */
     qTestEnvironmentVariables?: Transformer.EnvironmentVariable[];
+    /** Specify the path relative to enlistment root of the sources from which the test target is built */
+    testSourceDir?: string;

```

Comment:
>string [](start = 20, length = 6)

If this is a path, why do you make it as a string? We have Path in our DScript type.

Reply:
This is meant to be a relative path to a source folder. Is Path the right type to use here? What are the invariants guaranteed by the Path type? Will passing a relative path from enlistment root violate any of those invariants?

Reply:
@anirudhsanthiar Perhaps then make the type RelativePath...

callers can specify these at the callsite with.
```
r`my/folder`
```